torch/csrc/api/include/torch/nn/modules/sequential.h

```diff
@@ -312,9 +335,13 @@ class SequentialImpl : public Cloneable<SequentialImpl> {
 
   /// Adds a type-erased `AnyModule` to the `Sequential`.
   void push_back(AnyModule any_module) {
```

Comment:
Is this method still used now?

Reply:
It is still used by this line: https://github.com/pytorch/pytorch/blob/c3e3c5cc39165470ddab5afb6373399fdbd6598e/torch/csrc/api/include/torch/nn/modules/sequential.h#L111